web/src/views/RepoAdd.vue

```diff
@@ -13,13 +13,19 @@
       >
         <span class="text-wp-text-100">{{ repo.full_name }}</span>
         <span v-if="repo.active" class="ml-auto text-wp-text-alt-100">{{ $t('repo.enable.enabled') }}</span>
-        <Button
-          v-if="!repo.active"
-          class="ml-auto"
-          :text="$t('repo.enable.enable')"
-          :is-loading="isActivatingRepo && repoToActivate?.forge_remote_id === repo.forge_remote_id"
-          @click="activateRepo(repo)"
-        />
+        <div v-else class="ml-auto flex items-center">
+          <router-link
```

Comment:
If that's a link, should e format it as a link? E.g. https://github.com/woodpecker-ci/woodpecker/blob/main/web/src/components/user/UserAPITab.vue#L20

Reply:
I think all text links should look the same, and as we use it for swagger, forge links (PRs, refs, etc.) in the pipeline step list already I would prefer it.

Reply:
Just adding the hover class makes it not identifiable as a link :D users should see that this text is clickable, so I would add `text-wp-link-100` as well.

Reply:
I've done a lot of back and forth with the link color, but especially blue on black becomes an accessibility issue in terms of contrast quickly. Desaturating them might cause issues again, and for the color theme PR I decided to prioritize accessibility.